### PR TITLE
Make edge attributes work in DOT

### PR DESCRIPTION
**Pull request by ypkang**
DOT code like the following will successfully save the edge attribute in their context now
```
graph_root -> node_1 [edge=special, name=edge_1]
```

### CODE REVIEW

File: jaseci_core/jaseci/graph/edge.py [hunk 1]
```diff
@@ -123,6 +123,18 @@ def connects(self, source=None, target=None, ignore_direction=False):
                 return False
         return True
 
+    def set_context(self, ctx, arch=None):
```
Comment (ypkang) [inline jaseci_core/jaseci/graph/edge.py:126]: @marsninja I copied this function over from the node class. I am not 100% sure what the `arch` parameter is used for though...

File: jaseci_core/jaseci/jac/interpreter/architype_interp.py [hunk 2]
```diff
@@ -243,7 +244,7 @@ def run_dot_a_list(self, jac_ast):
             lhs_id = self.run_dot_id(kid[0])
             kid = kid[1:]
             if (len(kid) == 0 or kid[0].token_text() != '='):
-                # If there is rhs, treat it as a boolean value of True
+                # If there is no rhs, treat it as a boolean value of True
```
Comment (ypkang) [inline jaseci_core/jaseci/jac/interpreter/architype_interp.py:252]: Noticed this was a typo.

**Review by marsninja — APPROVED** [review]
Very nice leet haxorisim @ypkang

File: jaseci_core/jaseci/graph/edge.py (hunk 1 above)
Comment (marsninja) [reply jaseci_core/jaseci/graph/edge.py:126]: perfect, arch is for updated architypes if the edge itself is old.

**Pull request merged**